Sources/UILocalNotification+Sweet.swift

```diff
@@ -33,8 +46,21 @@
             return filteredNotifications.first
         }
 
-        public static func delete(_ id: String) {
-            if let notification = find(id) {
+
+        /// Cancels a local notification.
+        ///
+        /// - Parameter id: The locala notification id.
+        public static func cancel(using id: String) {
```

Comment:
Same here I would be worried that this `name` will show up for the user, i think the name of the parameter should make very clear that it's just a string to identify the notification with.